src/components/GearColumnProvider.js

```diff
@@ -29,14 +29,13 @@ export default function GearColumnProvider({ children, currentBook }) {
       {
         label: "Price",
         name: "price",
-        options: {
-          customBodyRender: (value, tableMeta) =>
-            `${
-              tableMeta.rowData[RESTRICTED_COL_INDEX] ? "(R) " : ""
-            }${value.toLocaleString()}`,
-        },
+        options: createRangeFilter("Price", true),
```

Comment:
If you check my branch (fix-price-sorting), I do this a bit differently. Instead of providing the entire options object, we should just provide the specific properties that are needed for the filter, then you can use the object spread operator to insert them into the existing options, that way the filter helper doesn't have to know anything about the column that it's giving filter options to. Like this:

`options: {
    someOption1: true,
    someOption2: "blablabla",
    ...getRangeFilterOptions()
}`